.github/workflows/linting.yaml

```diff
@@ -7,16 +7,25 @@ on:
   pull_request:
 
 jobs:
-  build:
+  lint:
     runs-on: ubuntu-latest
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
+      - name: Get modified src files
+        id: changed-src-files
+        uses: tj-actions/changed-files@v37
+        with:
+          files: src/*.c
```

Comment:
Don't you use .h files?

Reply:
First, one shouldn't lint header files with cppchek,
second, header files contain only function declarations and are used only for linking, so there's nothing to lint there.